source/features/search-results-filter.tsx

```diff
@@ -0,0 +1,90 @@
+import {React} from 'dom-chef/react';
+import select from 'select-dom';
+import features from '../libs/features';
+import domify from '../libs/domify';
+
+function cleanLinks(links) {
+	for (const link of links) {
+		const href = new URL(link.href);
+		href.searchParams.set('q', cleanSearchQuery(href.searchParams.get('q')));
+		link.href = href;
+	}
+}
+
+function getSearchQuery() {
+	return new URL(location.href).searchParams.get('q');
+}
+
+function cleanSearchQuery(query) {
+	return query.replace(/\s*\bis:(pr|issue)\b\s*/gi, '');
+}
+
+function createUrl(type) {
+	const query = cleanSearchQuery(getSearchQuery());
+	return `${location.origin}${location.pathname}?q=${encodeURIComponent(query)}+is%3A${type}&type=Issues`;
+}
+
+async function fetchCount(type) {
+	const response = await fetch(createUrl(type));
+	const text = await response.text();
+	const counter = domify(text).querySelector('.menu a.selected .Counter');
+	return counter ? counter.textContent : 0; // When there are no results, the counter is absent.
+}
+
+function prIsActive() {
+	return getSearchQuery().split(' ').includes('is:pr');
+}
+
+async function init() {
+	const menu = select('nav.menu');
+
+	const issueLink = select('a[href*="&type=Issues"]', menu);
+	if (!issueLink) { // Stop when issues are not enabled for this repo.
+		return;
+	}
+
+	const links = select.all('a', menu).filter(item => item !== issueLink);
+	cleanLinks(links);
```

Comment:
Since you're not using `links` anywhere else, you can move the `select` into `cleanLinks()`

Reply:
I need to filter out `issuesLink` from all links. What I did was remove the constant `links` and put the filter directly on the `cleanLinks` method.